### PR TITLE
Update 20191111.md

### PR DESCRIPTION
**Pull request by chartgerink**
Here are my notes for the call we just had. If approved, these turn into minutes 🎊

### CODE REVIEW

**Comment by dpaez** [comment]
@chartgerink LGTM I guess only needs to change the filename since this meeting happened on Monday 11th instead of our regular Tuesday meetings.

File: 20191111.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+-   Attending
+    - Chris
```
Comment (juliangruber) [inline 20191111.md]: add twitter / email links in case people want to follow individuals?

File: 20191111.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+-   [`@p2pcommons/specs`](https://github.com/p2pcommons/specs)
+    - Hyperdrive `v10` explicitly included in module spec now.
+    - [`dat.json` structure proposals](https://github.com/p2pcommons/specs/issues/8)
+        - What proposal has our preference?
+        - proposal 5
+        - [weblinks spec](https://html.spec.whatwg.org/multipage/semantics.html#the-link-element) concordance
+        - Chris will update the spec and double check with weblinks spec
+-   [`@p2pcommons/sdk-js`](https://github.com/p2pcommons/sdk-js)
+    - Update by Diego
+        - `0.2.8` -> fix missing statement
```
Comment (juliangruber) [inline 20191111.md]:
```suggestion
        - `0.2.8` -> fix missing `return` statement
```

doesn't really matter I guess but we might just

File: 20191111.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        - would like to spend more time with this
+        - we could look at flows for PGP keys for example
+        - usb keys?
+        - developments of hypertrie might be relevant? kappa also used (?)
+        - keep an eye on wireline, they might launch some stuff that we can use
+        - do we want to stay 100% decentralized and be unusable or introduce a tiny bit of centralization for usage?
+-   [`@hypergraph-xyz/cli`](https://github.com/hypergraph-xyz/cli)
+    - Update by Julian
+        - updated sdk to latest version, thanks Diego for patches :)
+        - CI environment is more stable, last two releases good immediately
```
Comment (juliangruber) [inline 20191111.md]:
```suggestion
        - CI environment is more stable, last two commits good immediately
```

**Pull request merged**